Nos/Views/Profile/ProfileHeader.swift

```diff
@@ -152,7 +152,7 @@ struct ProfileHeader: View {
             NavigationView {
                 BioSheet(author: author)
                     .background(Color.bioBgGradientBottom)
-                    .nosNavigationBar(title: .localizable.profile)
+                    .nosNavigationBar(title: .localizable.profileTitle, leadingPadding: 0)
```

Comment:
I don't love having to call this with `leadingPadding: 0`, but I found that there are around 21 uses of nosNavigationBar. If we defaulted the leadingPadding to 0, we'd have to add `leadingPadding: 14` to some of those calls. Although we actually have spacing problems in some of the other places, so maybe we should just update the default value.

Reply:
You fixed it while I was replying to your previous message xd I like it.
Yeah, its not cool, but given the context, I agree it is the best solution.

Reply:
Sorry to keep coming back to this PR, but I actually think we should remove the leading padding entirely, or at least default to 0. There are just so many screens that are off center right now, and since I'm already working on this, I'd like to fix them.

Reply:
OK, I've done this now. Everything is centered, and I think it all looks much better.